screenshots/screenshot.ps1

```diff
@@ -5,11 +5,14 @@
   Take a screenshot of the active window and output it to a PNG file.
 .PARAMETER FilePath
   The name of the file to write to.  The file will be a PNG.
+.PARAMETER Title
+  The title of the window to capture; if not give, defaults to the active window.
```

Comment:
if not given?

Reply:
Yeah; but I just dropped it, and made it _… window to capture; defaults to…_ instead.